src/@next/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../utilities/borderRadius';
+import { Neutral, Red } from '../utilities/colors';
+import { NotoSans } from '../utilities/fonts';
+
+export const StyledTextAreaContainer = styled.div`
+  position: relative;
+  width: 100%;
+  max-width: 520px;
+`;
+
+export const StyledTextArea = styled.textarea`
+  background: ${Neutral.B100};
+  box-sizing: border-box;
+  border: 1px solid ${Neutral.B68};
+  border-radius: ${borderRadius4};
+  padding: 8px 0px 25px 8px;
```

Comment:
Actually, the paddings of the text area should be 8px 12px, with two elements placed vertically inside
![image](https://github.com/glints-dev/glints-aries/assets/12844836/7ec41c94-ee9e-4977-a789-74f25406eedb)

Reply:
I used absolute positioning for the character count, but yea you're right it caused the problem above that text might overlap with the character counter because of this... fixing this issue now...